create.js

```diff
@@ -0,0 +1,10 @@
+const employee = {
+  name: 'ANE01',
+  email: 'Anna@example.com',
+  phone: '01123456789',
+  isMale: true,
+};
+
+const name = employee.name;
```

Comment:
Hey man, you can destructure out each key value as a variable from the object.  Good job doe!
const { name, email, phone } = employee;